enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/OCINode.java

```diff
@@ -111,15 +112,21 @@ public static final List<? extends Action> actionsForPath(String path, Lookup lk
     }
     
     public void refresh() {
-        if (factory != null) {
-            factory.refreshKeys();
-        }
-        update(item);
+        RequestProcessor.getDefault().post(() -> {
```

Comment:
Note: the default RP (see the documentation for getDefault()) has a guard against 'too often scheduled task'. Not sure if "refresh" on a parent node (that contains children) refreshes the entire subtree one by one - that may cause the guard to trigger & print some annoying messages into the log. If the refresh only affects a single node, it's fine.

Reply:
Just tested it, refresh() method will be triggered only once, no matter the node hierarchy